lib/string_reverse.rb

```diff
@@ -1,4 +1,19 @@
 # A method to reverse a string in place.
 def string_reverse(my_string)
-  my_string << "not implemented"
+  # my_string << "not implemented"
+
+  return nil if my_string == nil
+
+  length = my_string.length
+  first_place = 0
+  last_place = (length -1)
+
+  while first_place < last_place
+    temp = 0

```

Comment:
Minor: You can safely delete line 12 since it's not adding any value to the logic you're trying to implement.